src/run-context/runContext.ts

```diff
@@ -55,9 +49,10 @@ export class RunContext {
   // apply to the source and the operand 0.
   computeOp1Address(
     op1Src: Op1Src,
-    op1Offset: Int16,
+    op1Offset: number,
     op0: MaybeRelocatable | undefined
-  ): Result<Relocatable> {
+  ): Relocatable {
+    SignedInteger16.ensureInt16(op1Offset);
```

Comment:
I don't know if removing all the primitives type is a good idea. Here refactoring was easy because you know from before (because it was typed before) that you needed to add this `ensureInt16`. Could we keep the `Int16`, but have it be just a number? So just use type aliasing? Would make it more clear also for the reader and the dev imo